order.js

```diff
@@ -70,20 +88,27 @@ module.exports = {
       'grid-row-gap',
       'columns',
       'column-count',
+      'column-fill',
       'column-width',
+      'column-span',
```

Comment:
why not above width?

Reply:
what is the rule here?

Reply:
> what is the rule here?

https://developer.mozilla.org/en-US/docs/Web/CSS/column-span